crates/fortitude_linter/src/rules/mod.rs

```diff
@@ -118,6 +118,7 @@ pub fn code_to_rule(category: Category, code: &str) -> Option<(RuleGroup, Rule)>
         (Modernisation, "011") => (RuleGroup::Stable, Ast, Default, modernisation::old_style_array_literal::OldStyleArrayLiteral),
         (Modernisation, "021") => (RuleGroup::Stable, Ast, Default, modernisation::relational_operators::DeprecatedRelationalOperator),
         (Modernisation, "031") => (RuleGroup::Preview, Ast, Optional, modernisation::include_statement::IncludeStatement),
+        (Modernisation, "041") => (RuleGroup::Preview, Ast, Optional, modernisation::out_of_line_attribute::OutOfLineAttribute),
```

Comment:
I'd argue that this should be part of the default set, I would hope that this would be fairly uncontroversial.

```suggestion
        (Modernisation, "041") => (RuleGroup::Preview, Ast, Default, modernisation::out_of_line_attribute::OutOfLineAttribute),
```